.github/fixtures/test-date-order-arg/cliff.toml

```diff
@@ -0,0 +1,60 @@
+# configuration file for git-cliff (0.1.0)
```

Comment:
Can you make the configuration files in this PR more minimal? Default values are now used when you don't specify fields. See #45

Reply:
Sure, applied in e7210a7.

Reply:
Should I use `Feat` or should I change the default settings to `Features` so that it matches `examples/cliff.toml`?

![image](https://user-images.githubusercontent.com/31987104/152726636-222c851a-469b-4a66-9a85-63f1c72e7b74.png)

Reply:
By the way, what is the default behavior? No. 1?

1. Capitalize the `type`
2. Load `example/cliff.toml` and use it
3. Else

Reply:
I think you can leave it as `Feat` here. There is no need to complicate it.

Default behavior is: scope is capitalized if no `commit_parsers` are used to override it. Is that what you asked?

Reply:
Yes, thank you! I'll fix it.